COLLABORATOR_GUIDE.md

```diff
@@ -127,7 +127,7 @@ can be fast-tracked and may be landed after a shorter delay:
 * Focused changes that affect only documentation and/or the test suite.
   `code-and-learn` and `good-first-issue` pull requests typically fall
   into this category.
-* Changes that revert commit(s) and/or fix regressions.
+* Changes that revert recently landed commit(s) and/or fix regressions.
```

Comment:
Just "Changes that fix regressions"?  That covers reverts.

It doesn't matter how old the commit is.  If it introduces a regression that can't be fixed speedily, first order of business should be to revert.

Reply:
Ack